src/App.tsx

```diff
@@ -1,27 +1,74 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { GoodsList } from './GoodsList';
 
-// import { getAll, get5First, getRed } from './api/goods';
-// or
-// import * as goodsAPI from './api/goods';
+import { Good } from './types/Good';
+import { getAll, get5First, getRed } from './api/goods';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <h1>Dynamic list of Goods</h1>
+export const App: React.FC = () => {
+  const [goods, setGoods] = useState<Good[]>([]);
+  const [error, setError] = useState('');
 
-    <button type="button" data-cy="all-button">
-      Load all goods
-    </button>
+  const handleGetAll = async () => {
+    try {
+      const goodsFromServer = await getAll();
 
-    <button type="button" data-cy="first-five-button">
-      Load 5 first goods
-    </button>
+      setGoods(goodsFromServer);
+    } catch {
+      setError('fucking Error!');
```

Comment:
![excuse-me-sir-3683865ca8](https://user-images.githubusercontent.com/102182808/221149969-c731a1e4-ee7a-49f3-8f61-e8ee21cdb01c.jpg)

Reply:
Sorry, I was very emotional when I wrote this code...